assets/test/protractor.conf.js

```diff
@@ -190,10 +190,10 @@ exports.config = {
   // The timeout in milliseconds for each script run on the browser. This should
   // be longer than the maximum time your application needs to stabilize between
   // tasks.
-  allScriptsTimeout: 11000,
+  allScriptsTimeout: 60000,
 
   // How long to wait for a page to load.
-  getPageTimeout: 10000,
+  getPageTimeout: 30000,
```

Comment:
After the final step of the creation flow, right after [this](https://github.com/openshift/origin/pull/3999/files#diff-c76167480ada997df0b3ae76b42ed923R113). The actual reason why this was timing out wasn't completely clear, but I think it has to do with the lack of support of websockets in our version of PhantomJS.